localstack/services/apigateway/apigateway_listener.py

```diff
@@ -261,25 +262,33 @@ def invoke_rest_api(api_id, stage, method, invocation_path, data, headers, path=
             # Sample request context:
             # https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-lambda.html#api-gateway-create-api-as-simple-proxy-for-lambda-test
             request_context = {
-                'path': relative_path,
+                # adding stage to the request context path.
+                # https://github.com/localstack/localstack/issues/2210
+                'path': '/' + stage + relative_path,
                 'accountId': account_id,
                 'resourceId': resource.get('id'),
                 'stage': stage,
                 'identity': {
                     'accountId': account_id,
                     'sourceIp': source_ip,
                     'userAgent': headers['User-Agent'],
-                }
+                },
+                'httpMethod': method,
+                'protocol': 'HTTP/1.1',
+                'requestTime': datetime.datetime.utcnow(),
+                'requestTimeEpoch': time.time() * 1000,
```

Comment:
this appears to be breaking integration with go lambdas

```
2020-06-18T16:40:29:WARNING:localstack.services.awslambda.lambda_api: Error executing Lambda function arn:aws:lambda:us-east-1:000000000000:function:csp-report-api-local-post: Lambda process returned error status code: 1. Result: {"errorType":"UnmarshalTypeError","errorMessage":"json: cannot unmarshal number 1592498428607.922 into Go struct field APIGatewayProxyRequestContext.requestContext.requestTimeEpoch of type int64"}
```

may need to cast to int/long?